src/classes/Inventory.ts

```diff
@@ -195,6 +204,21 @@ export default class Inventory {
         return this.findBySKU(sku, tradableOnly).length;
     }
 
+    getAmountOfGenerics(sku: string, tradableOnly?: boolean): number {
+        const s = SKU.fromString(sku);
+        if (s.quality === 5) {
+            // generic getAmount so return total that match the generic sku type
+            return this.unusualEffects
+                .map(e => {
+                    s.effect = e.id;
+                    return this.getAmount(SKU.fromObject(s), tradableOnly);
+                })
+                .reduce((p, c) => (p ? p + c : c));
```

Comment:
Same comment as above (potential refactoring of variable names here for clarity)

Reply:
Updated